rush/rush-lib/src/rush-schema.json

```diff
@@ -100,6 +100,16 @@
         "additionalProperties": false,
         "required": [ "packageName", "projectFolder" ]
       }
+    },
+    "pinnedVersions": {
+      "description": "A mapping of dependency package names to a semantic version. These are included in the common package.json and is used to control versions of second-level dependencies.",
+      "type": "object",
+      "patternProperties": {
+        "^[a-zA-Z0-9_@/-]+$": {
```

Comment:
Is this RegExp supposed to describe a package name?  The actual rules are [fairly complicated](https://github.com/npm/validate-npm-package-name/blob/master/index.js).  Maybe it would be better to avoid trying to catch these errors during schema validation.  You could enforce something simple like `"^\\S+$"`.  Don't forget to escape the backslash